cache/save_test.go

```diff
@@ -74,12 +74,12 @@ func Test_ProcessSaveConfig(t *testing.T) {
 			input: SaveCacheInput{
 				Verbose: false,
 				Key:     "cache-key",
-				Paths:   []string{"~/.bash_h*"},
```

Comment:
This is a bit of an ugly test case as it relies on "the outside world", and it changed recently (there is no `.bash_history` file on the VMs). But there is no other way to test the tilde resolution, I hope the `.ssh` folder won't go away any time soon 😅